.storybook/vanillaish.js

```diff
@@ -3,33 +3,15 @@ import { create } from "@storybook/theming/create";
 export default create({
   base: "light",
 
-  colorPrimary: "#fff",
-  colorSecondary: "rgba(0,0,0,0.05)",
-
   // UI
-  appBg: "#fff",
-  appContentBg: "#fff",
-  appBorderColor: "#cdcdcd",
   appBorderRadius: ".125rem",
 
   // Typography
   fontBase:
     '"Ubuntu", -apple-system, "Segoe UI", "Roboto", "Oxygen", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif',
   fontCode: '"Ubuntu Mono", Consolas, Monaco, Courier, monospace',
 
-  // Text colors
-  textColor: "#111",
-  textInverseColor: "#111",
-
-  // Toolbar default and active colors
-  barTextColor: "#111",
-  barSelectedColor: "#666",
-  barBg: "#fff",
-
   // Form colors
-  inputBg: "#fff",
-  inputBorder: "#cdcdcd",
-  inputTextColor: "#111",
   inputBorderRadius: ".125rem",
```

Comment:
Same here, we probably don't need that rounded corner.

Reply:
The default is much more round, I think better to keep it.